src/bin/bat/main.rs

```diff
@@ -118,11 +120,17 @@ pub fn list_languages(config: &Config) -> Result<()> {
     Ok(())
 }
 
+fn theme_preview_file<'a>() -> Input<'a> {
+    Input::from_reader(Box::new(BufReader::new(THEME_PREVIEW_DATA)))
+        .with_name(Some("theme.rs".as_ref()))
```

Comment:
This is now needed for syntax detection?

Reply:
Not particularly. I'll remove it